doc/md_src_files/082_relatedSystems.md

```diff
@@ -105,3 +105,74 @@ In that sense, it is compatible with any of methods mentioned above or SSVC.
+This section provides an overview of some options; we cannot claim it is exhaustive.
+Each decision point has a subsection for `Gathering Information About` it.
+These sections provide suggestions that would also contribute to creating or honing information feeds. 
+However, if there is a viable information source we have not captured, please create an [issue on the SSVC GitHub page](https://github.com/CERTCC/SSVC/issues) explaining the information source and what decision point it informs.  
+
+Various vendors provide paid feeds of vulnerabilities that are currently exploited by attacker groups.
+Any of these could be used to indicate that [*active*](#exploitation) is true for a vulnerability.
+Although the lists are all different, we expect they are all valid information sources to be used for this purpose.
+Although we are not aware of a comparative study of the different lists of active exploits, we expect they have similar properties to block lists of network touchpoints [@metcalf2015blocklist] and malware [@kuhrer2014paint].
+Namely, each list has a different view and vantage on the problem, which makes them appear to be different, but each list does a good job at accurately representing its particular vantage at a point in time.
+
+
+[*System Exposure*](#system-exposure) should be informed by the various scanning platforms such as and Shodan and Shadowserver.
+A service on a device should be scored as [*open*](#system-exposure) if such a general purpose Internet scan finds that the service responds.
+Such scans do not find all [*open*](#system-exposure) systems, but any system they find should be considered [*open*](#system-exposure).
+Scanning software, such as the open-source Nessus, could be used to scan for connectivity inside an organization to catalogue what devices should be scored [*controlled*](#system-exposure) if, say, the scan finds them on an internal network where devices regularly connect to the Internet.  
+
+
+There are some information sources that are not exactly suited to make SSVC decisions, but with a bit of work could be used directly.
+Three prominent examples of this are CVSS impact base metrics, CWE, and CPE.
+
+[*Technical Impact*](#technical-impact) is directly related to the CVSS impact metric group.
+However, this metric group cannot be directly mapped to [*Technical Impact*](#technical-impact) in CVSSv3  because of the Scope metric.
+[*Technical Impact*](#technical-impact) is only about adversary control of the vulnerable component.
+If the value of Scope is Changed in CVSSv3, then the impact metrics are the maximum of the impact on the vulnerable component and other components in the environment.
+As long as the impact metrics in CVSS are clearly about just the vulnerable component, if confidentiality, integrity, and availability metrics are all "high" then [*Technical Impact*](#technical-impact) is [*total*](#technical-impact).
```

Comment:
Move the "as long as" clause to the end of the sentence? That's a lot to hold in memory until you know what to apply it to.